### PR TITLE
Fix current statistics and contrasts in code-coverage-plugin

### PR DESCRIPTION
**Pull request by ganhammar**
## Hey, I just made a Pull Request!

I'm sorry, this is a rather big pull request, a lot of it is just to get the dev server running. Other than that, this pull request tries to address the following:

* There was a bug in the current statistics, the number shown is the oldest record rather than the latest
* Removed the additional Paper around the "Explore Files" table
* Add icons in front of folders as well as files in the file explorer, to make it clearer what the row represents
* Switched the highlighter theme to a theme that provides somewhat better contrasts against the backgrounds

Hope this is okay! Thanks!

### Screenshots

Before (light theme):
Screenshot 2023-09-20 at 22 26 11
Screenshot 2023-09-20 at 22 27 09

After (light theme):
Screenshot 2023-09-20 at 22 32 16
Screenshot 2023-09-20 at 22 32 46

Before (dark theme):
Screenshot 2023-09-20 at 22 26 25
Screenshot 2023-09-20 at 22 26 49

After (dark theme):
Screenshot 2023-09-20 at 22 32 34
Screenshot 2023-09-20 at 22 32 58

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jhaals** [comment]
Nice work!
It definitely looks a lot better, thanks for attaching the screenshots, it really helps! 👍
CC @alde to take a peak at this too

**Comment by ganhammar** [comment]
Thanks @jhaals! Sorry for doing this, I know you're not a default reviewer for [this](https://github.com/backstage/backstage/pull/19980) PR, but, alde has reviewed it, any chance it could get a push? We have several teams that depend on it 😊

File: .changeset/stupid-eagles-shave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-code-coverage': patch
+---
+
+Updated layout to improve contrasts and consistency with other plugins
```
Comment (alde) [inline .changeset/stupid-eagles-shave.md:5]: that's not the only change, would it be a lot to ask you to split it into two PRs? One for visuals and one to fix the bug?

Comment (ganhammar) [reply .changeset/stupid-eagles-shave.md:5]: You are right, that makes a lot of sense! I'll look into that!

Comment (ganhammar) [reply .changeset/stupid-eagles-shave.md:5]: Okay @alde, fixed now! I'll create a new PR with the fix once this PR has been reviewed and merged. _Note that print screens are not updated._

**Review by alde — APPROVED** [review]

**Pull request merged**